README.md

```diff
@@ -82,6 +88,24 @@ kubefirst init \
 --hosted-zone-name domain.example
 ```
 
+### GitHub
+
+To prepare the installation using GitHub you can:
+
+```bash
+kubefirst init \
+--admin-email email@example.com.com \
+--cloud aws \
+--hosted-zone-name example.com \
+--region eu-central-1 \
+--cluster-name example_com \
+--profile default \
+--github-user example-github-username \
+--github-org example-organization \
+--gitops-branch main \
```

Comment:
this choices are not expected for users to use, unless a bug is detected..

Reply:
`--gitops-branch` and `--metaphor-branch main`

Reply:
few days ago I tried without these flags and installation failed. we could merge it like this, and update asap the new tags are created to fix this.

Reply:
it will only work, once we tag 1.9.0, until there we need to pass it.

Reply:
hey fellas, we can course correct in the morning, these docs won't publish to prod until we're ready, but getting prepped for prod and it looks like i want these in. forgive me and help me adjust if i'm overstepping with the merge and we can adjust before prod publish. @converge @6za